flex/engines/graph_db/grin/src/predefine.h

```diff
@@ -0,0 +1,66 @@
+#include "grin/predefine.h"
+
+#include "storages/rt_mutable_graph/mutable_property_fragment.h"
+
+typedef gs::oid_t GRIN_OID_T;
+typedef gs::vid_t GRIN_VID_T;
+
+typedef gs::MutablePropertyFragment GRIN_GRAPH_T;
+
+typedef struct GRIN_VERTEX_T {
```

Comment:
[future optimize] use a int64 to encode vertex will eliminate heap allocations, which might be more efficient